app/src/main/res/layout/listview_tent_header.xml

```diff
@@ -24,6 +24,6 @@
         android:layout_height="wrap_content"
         android:text="@string/unknown_tent"
         android:textAppearance="@style/text.heading"
-        tools:text="@string/probable" />
+        tools:text="Probable 1 (10 patients)" />
```

Comment:
The tools:text string is never shown to users -- it only appears as sample
text in the graphical UI layout editor in Android Studio.
On Oct 13, 2015 3:49 PM, "viniciusboson" <notifications@github.com> wrote:

> In app/src/main/res/layout/listview_tent_header.xml
> <https://github.com/projectbuendia/client/pull/78#discussion_r41875659>:
>
> > @@ -24,6 +24,6 @@
> >          android:layout_height="wrap_content"
> >          android:text="@string/unknown_tent"
> >          android:textAppearance="@style/text.heading"
> > -        tools:text="@string/probable" />
> > +        tools:text="Probable 1 (10 patients)" />
>
> Shouldn't it be internationalized ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projectbuendia/client/pull/78/files#r41875659>.
>